### PR TITLE
Dt microservice

### PR DESCRIPTION
**Pull request by DavidTTran**
Completed
- Write tests for each of our routes
- Try to set up DB for serializers and caching
   - I don't think caching works for our needs because it requires a database, and we're never actually saving a recipe

### CODE REVIEW

File: app/controllers/recipes_controller.rb [hunk 1]
```diff
@@ -1,10 +1,11 @@
 require './app/services/spoonacular_service'
-require './app/serializers/recipe_serializer'
+require 'sinatra'
 
 class RecipesController < Sinatra::Base
 
   get '/recipes/complexSearch' do
     SpoonacularService.new.complex_search(params).to_json
+    # RecipeSerializer.new(recipes)
```
Comment (mikez321) [inline app/controllers/recipes_controller.rb:8]: probably a line we can take out later on

File: spec/requests/spoonacular_spec.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    request = get "/recipes?ids=1,2,3"
+    response = JSON.parse(request.body, symbolize_names: true)
+    assert_equal 3, response.size
+
+    ids = response.reduce([]) do |acc, recipe|
+      acc << recipe[:id]
+    end
+    assert_equal [1,2,3], ids
+  end
+end
```
Comment (alex-latham) [inline spec/requests/spoonacular_spec.rb:37]: Tests! Woohoo! I miss Minitest a little bit.

**Review by mikez321 — APPROVED** [review]
Aside from the pry/unused method and commented line, the rest of it is looking good.  Its fun to see minitest sometimes, makes me remember the good ol days!

**Review by alex-latham — APPROVED** [review]
Minor changes requested, but not blocking approval.

**Review by alex-latham — APPROVED** [review]

**Pull request merged**